sdk/python/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "openlit"
-version = "1.29.0"
+version = "1.29.1"
```

```suggestion
version = "1.29.2"
```
My bad, just released 1.29.1 so we can use `1.29.2` here